ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -175,7 +174,8 @@
   </p>
 
   <p id="example-size">
-    <strong><a href="#example-size" class="ds-a" rel="external">&sect;</a>Size:</strong>
+    <strong><a href="#example-size" class="ds-a" rel="external">&sect;</a>
```

Comment:
some UI polish

WAS:
![image](https://user-images.githubusercontent.com/1372946/134437593-70e7a386-a190-42f8-b4a5-2ff22eb763c9.png)

IS:
![image](https://user-images.githubusercontent.com/1372946/134437622-3c5c18f6-237e-486d-9f86-95ac2002b989.png)